corehq/apps/domain/views.py

```diff
@@ -1825,8 +1826,8 @@ def post(self, request, *args, **kwargs):
                     copy_by_id.add(k[:-len("-publish")])
 
             old = self.domain_object.published_snapshot()
-            new_domain = self.domain_object.save_snapshot(ignore=ignore,
-                                                          copy_by_id=copy_by_id)
+            new_domain = self.domain_object.save_snapshot(
+                share_reminders=share_reminders, copy_by_id=copy_by_id)
```

Comment:
Yeah I pushed this down into `save_snapshot`. Since this is the only call to it and always would pass in `share_user_roles=False`, I just resolved not to have that param at all and have `save_snapshot` always call `save_copy` with `share_user_roles=False`